docs/architecture-cli.md

```diff
@@ -0,0 +1,133 @@
+---
+title: The Talk cli
+keywords: architecture
+sidebar: talk_sidebar
+permalink: architecture-cli.html
+summary:
+---
+
+Talk ships with a cli tool that allows access to a wide variety of functionality.
+
+It is designed to provide a convenient way for engineers to perform key tasks without the need to muck about in the UI. It also opens the door for scripts to perform operations programmatically.
+
+Note: the cli tool requires [the Talk environment to be configured](configuration.html) either via env vars or by using a `.cli` file via `bin/cli -c .env [command] ....`
+
+## Using the cli
+
+In a terminal, try:
+
+```
+/path/to/talk/bin cli [options] [commands] [arguments]
```

Comment:
You can also note that in the Docker environment, `cli` is in the PATH, so you can just type `cli ...`